src/closure.js

```diff
@@ -5,15 +5,41 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  let count = 0;
+  const newCounter = () => {
+    return ++count;
+  };
+  return newCounter;
 };
 
 const counterFactory = () => {
+  let count = 0;
+  const obj = {
+    increment() {
+      count++;
```

Comment:
Here you can just do `return count++;` and `return count--;` for these functions.